src/EditorFeatures/Core/Structure/AbstractStructureTaggerProvider.cs

```diff
@@ -206,9 +206,9 @@ protected sealed override ITaggerEventSource CreateEventSource(ITextView? textVi
         ImmutableArray<BlockSpan> spans)
     {
         var snapshot = snapshotSpan.Snapshot;
-        spans = GetMultiLineRegions(outliningService, spans, snapshot);
+        var multiLineSpans = GetMultiLineRegions(outliningService, spans, snapshot);
```

Comment:
The number of entries I see returned by this method for large files exceeds 128, so directly using an ArrayBuilder isn't a great idea. I figured since this a private method with a single user that just enumerates over the results, this was acceptable.

Reply:
can you doc that then.  it would be very easy for this to get converted back.  Also, we can just have our own pool here.  but i'm good either way.

Reply:
done!